src/backend/wine/manager/utils.ts

```diff
@@ -153,11 +173,11 @@ async function installWineVersion(
     }
   } catch (error) {
     if (abortController.signal.aborted) {
-      logWarning(error, LogPrefix.WineDownloader)
-      return 'abort'
+      logWarning(String(error), LogPrefix.WineDownloader)
+      return 'abort' as const
```

Comment:
These look unrelated, `as const` isn't going to do anything here, and the logger can handle `error` itself just fine (it's gonna do a better job at stringification than `String` at least)
Note that there are other instances of this further down

Reply:
same, just typescript warnings